tests/pkcs1v15.rs

```diff
@@ -0,0 +1,51 @@
+use pkcs8::DecodePrivateKey;
+use signature::Signer;
+
+use rsa::pkcs1v15::SigningKey;
+use rsa::RsaPrivateKey;
+
+const RSA_PRIV_PEM: &str = r#"-----BEGIN PRIVATE KEY-----
```

Comment:
We already have private key fixtures under https://github.com/RustCrypto/RSA/tree/master/tests/examples/pkcs8 which you can import with `include_str!`

Reply:
OK. Let me try to gen the test vector with the program above and the predefined private key.